test/bucklescript/src/__snapshots__/index_test.bs.js.snap

```diff
@@ -32,3 +32,12 @@ exports[`ComponentLink should render an <a /> tag 1`] = `
   />
 </div>
 `;
+
+exports[`ComponentWithParameter renders 1`] = `
+<div>
+  <div
+    class="css-1roeevf"
+    color="5194459,FF0000"
```

Comment:
As @davesnx mentioned that's not desirable, but will merge it because that's non blocking